doc/use_cases/turtlebot3_tutorial.md

```diff
@@ -0,0 +1,239 @@
+
+ROS2 Navigation2 - Turtlebot3 - Oct, 2019
+# Navigation2 - Turtlebot 3
+
+- [Overview](#overview)
+- [Requirments](#requirements)
+- [Navigation2 with Turtlebot 3 in Gazebo](navigation2-with-turtlebot-3-in-gazebo)
+- [Navigation2 with a Real Turtlebot 3](navigation2-with-a-real-turtlebot-3)
+- [Videos](#videos)
+
+# Overview:
+
+This tutorial shows how to control and navigate Turtlebot 3 using the ROS2 Navigation2. Turtlebot is a low-cost, personal robot kit with open-source software (ROS2). Turtblebot robots are widely supported by the ROS community. You can find many applications and examples of Turtlebot projects on the Internet. You can find more information about Turtlebot3 [here.](http://emanual.robotis.com/docs/en/platform/turtlebot3/overview/)
+
+Links to the robot kits:
+
+- [Turtlebot Burger](http://www.robotis.us/turtlebot-3-burger-us/)
+- [Turtlebot Waffle](http://www.robotis.us/turtlebot-3-waffle-pi/)
+
+This tutorial consists of two parts. In the first part, you will learn how to use Turtlebot 3 robots in summation (Gazebo). And in the second part, you will learn how to control a real Turtlebot Waffle using Navigation2.
+
+```ROS2 Dashing``` and ```Navigation2 Dashing 0.2.4``` are used to create this tutorial.
+
+This tutorial may take about 2-5 hours to complete completely depends on your experience with ROS, robots and what computer system you have.
+
+## Requirements:
+
+- [Install ROS2]([https://index.ros.org/doc/ros2/Installation/](https://index.ros.org/doc/ros2/Installation/))
+
+- Install Navigation2
+
+    - `sudo apt install ros-<ros2-distro>-navigation2`
+
+    - `sudo apt install ros-<ros2-distro>-ros-dashing-nav2-bringup`
+
+- [Build Turtlebot3](http://emanual.robotis.com/docs/en/platform/turtlebot3/ros2/#setup)
+
+- [Install Gazebo](http://gazebosim.org/tutorials?tut=install_ubuntu&cat=install)
```

Comment:
if they did a desktop install, they don't need this. Do you want to recommend desktop install? I'd think if someone is reading this they better just go with that as the default.

Reply:
I haven't tested Turtlebot3 via apt yet, but it should work :).

We don't know what ROS packages the user has on their system, so I left this step in the requirements.

Reply:
I was pointing to gazebo, to be specific which will come with a desktop install

Reply:
OK Gazebo install instruction is gone